src/lib/pickles/pseudo/pseudo.ml

```diff
@@ -42,22 +43,6 @@ module Make (Impl : Snarky_backendless.Snark_intf.Run) = struct
   module Domain = struct
     let num_shifts = Nat.to_int Pickles_types.Plonk_types.Permuts.n
 
-    (** Compute the 'shifts' used by the kimchi permutation argument.
-
-        These values are selected deterministically-randomly to appear outside
```

Comment:
This part of the comment is relevant to the implementation; it should probably live here as a non-doc-comment, rather than in the interface